src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -110,6 +110,18 @@ public DashboardWebApplication(Action<WebApplicationBuilder>? configureBuilder =
         builder.Services.AddLocalization();
 
         _app = builder.Build();
+
+        // this needs to be explicitly enumerated for each supported language
+        // our language list comes from https://github.com/dotnet/arcade/blob/89008f339a79931cc49c739e9dbc1a27c608b379/src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.props#L22
+        var supportedLanguages = new[]
+        {
+            "en", "cs", "de", "es", "fr", "it", "ja", "ko", "pl", "pt-BR", "ru", "tr", "zh-Hans", "zh-Hant"
+        };
+
+        _app.UseRequestLocalization(new RequestLocalizationOptions()
+            .AddSupportedCultures(supportedLanguages)
```

Comment:
It's possible we could support more format cultures than UI cultures but don't block on that. It might be something to consider later if we get requests as this is the reason these are split. UI cultures are about loading resources (typically strings) whereas the general culture is used to format values, calendar-related operations, etc.